package.json

```diff
@@ -13,7 +13,16 @@
     "preview": "vite preview",
     "test": "vitest",
     "test-run": "vitest run",
-    "coverage": "vitest run --coverage"
+    "coverage": "vitest run --coverage",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:check": "eslint",
+    "lint:file": "eslint --fix",
```

Comment:
`lint:file` would try to autofix, I found the name a bit confusing, should it be `lint:fix` since it'd fix all vue files?